style.css

```diff
@@ -82,3 +101,360 @@
   padding-left: 0;
   padding-top: 20px;
 }
+
+.works {
+  display: flex;
```

Comment:
- Great Job for implementing previous reviewer suggestions:+1: however you messed updating `display:flex` to `display:grid` please update it AS  it is part of the [requirent](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)